game/world/managers/objects/units/player/StatManager.py

```diff
@@ -850,6 +879,21 @@ def get_mana_bonus_from_intellect(intellect):
     Classes.CLASS_WARRIOR: (3.9, 20.0)
 }
 
+# VMaNGOS (level 1, level 60, agility)
+# 0.5.3 Strength improve critical strike
+# TODO: THIS IS A GUESS, find the real scale for strength
```

Comment:
I predict this is one of those TODOs that will never be solved, since there is no way to gather alpha data about this. And by the way, the scaling in vmangos isn't exactly right, it's just an approximation that is off at low levels. I was planning to change it into a lookup table, with the crit per agility for each level and class, and fill in data from sniffs, then extrapolate the missing level and class combinations.